.github/workflows/main.yml

```
@@ -36,7 +36,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        ruby-version: ["2.6", "2.7", "3.0", "3.1", "3.2"]
+        ruby-version: ["2.6", "2.7", "3.0", "3.1", "3.2", "3.3""]
```

actionlint (1.7.3)

could not parse as YAML: yaml: line 39: found unexpected end of stream

(syntax-check)